maintainers/maintainer-list.nix

```diff
@@ -14,6 +14,37 @@
      keys = [{
        fingerprint = "AAAA BBBB CCCC DDDD EEEE  FFFF 0000 1111 2222 3333";
      }];
+
+     # Default null; should be a free-form string explaining why the maintainer
+     # became inactive.
+     #
+     # The most immediate use of this attribute is to mark maintainers that are
+     # not contributing to Nixpkgs from a long amount of time.
+     # Nonetheless it can be used to mark any *long term inactivity*, including
+     # but not limited to:
+     #
+     # - Sabbatical leave
+     # - Retirement
+     # - Intimate issues
```

Comment:
... what?

Reply:
I guess they meant "Private issues" or whatever fallback when you don't feel like providing reasons?

Reply:
Bad translation.
The idea is "any personal issue I don't want to talk about". From rehab to mourning.